andela/ah-jarvis
# Ft article tags mockup 159054019

**What does this PR do?**
Create the Mockup design for Tags

**Description of Task to be completed?**
Author should be able to add tags
**How should this be manually tested**
Prerequisites
You should have sass and live-server installed

`clone this project`
Run the following commands
`cd ah-jarvis`

`git fetch`

`ft-article-tags-mockup-159054019`
  
`sass ui/sass/main.scss ui/static/styles.css`

`live-server .`
The live server will open a page in the browser
Navigate to `http://127.0.0.1:8080/ui/public/article/create/`
Screenshots
<img width="1280" alt="screen shot 2018-08-17 at 12 19 47" src="https://user-images.githubusercontent.com/18376530/44258885-b1c20f00-a218-11e8-9050-1935e7a33065.png">



What are the relevant pivotal tracker stories?
[#159054019](https://www.pivotaltracker.com/story/show/159054019)
